README.md

```diff
@@ -24,14 +32,73 @@ On the other hand, if BO information is collected in a central register via othe
             "scheme": "PRY-IDCARD",
             "id": "12345"
           },
-          "nationality": "PY"
+          "nationality": "PY",
+          "address": {
+            "streetAddress": "Avenida Eusebio Ayala 1347",
+            "locality": "Asunción",
+            "region": "Gran Asunción",
+            "postalCode": "1001",
+            "countryName": "Paraguay"
+          },
+          "email": "jperez@example.com",
+          "faxNumber": "+595210000001",
+          "telephone": "+595210000000"
         }
       ]
     }
   ]
 }
 ```
 
+### Without nationality
+
+The beneficial owner's nationality is not disclosed because the organization is listed on a regulated market (e.g. a stock exchange) that ensures adequate transparency of the nationality of beneficial owners.
```

Comment:
```suggestion
The beneficial owner's nationality is not disclosed because the organization is listed on a regulated market (e.g. a stock exchange) that ensures adequate transparency in line with anti-money laundering legislation.
```